build.gradle

```diff
@@ -244,14 +244,20 @@ task dockerCopyFiles(type: Copy) {
 task dockerBuildImage(type: DockerBuildImage) {
     dependsOn 'dockerCopyFiles'
     inputDir = project.file('build/docker/')
-    tag = "${project.dockerTagBase}/${jar.baseName}:${project.version}"
+    tags.add("${project.dockerTagBase}/${jar.baseName}:${project.version}".toString())
+    tags.add("${project.dockerTagBase}/${jar.baseName}:latest".toString())
 }
```

Comment:
This preserves the old tagging of images with snapshot versions (`:5.0.2-SNAPSHOT`) as well as tags the latest image with `:latest`

Reply:
Is there something that uses the snapshot docker images?  @bdrainer , if this isn't used can Will just remove it?